### ISSUE TITLE
Sidebar: fix styling

### ISSUE THREAD

**Issue by AnnaSolovykh** [JULIERAJ/KIDS-FIRST-Front#13]
[Figma](https://www.figma.com/design/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=501-3083&t=eLLYUujZyqokFEVQ-0)
1) Fix `Selected Hover` in Sidebar - when the tab in active, when it is selected, this styling should be applied
2) Add focus styling for accessibility (when we use arrows on the keyboard, the `Focus` styling  should be added)

![Image](https://github.com/JULIERAJ/KIDS-FIRST-Front/assets/114008959/e8a9b295-81c6-4671-a1fa-3c9eaaf2e6d2)

**Comment by JULIERAJ** [comment]
Closing issue. Was market as  ✅ Completed in [KIDS FIRST MVP 2024] project tracker.
